openstates/la/legislators.py

```diff
@@ -102,32 +93,29 @@ def scrape_upper(self, chamber, term):
 
     def scrape_lower_legislator(self, url, leg_info, term):
         page = self.lxmlize(url)
+
+        name = page.xpath('//div[@class="FullName"]/text()')[0].strip()
+        if name.startswith("District ") or name.startswith("Vacant "):
+            self.warning("Seat is vacant: {}".format(name))
+            return
+
         photo = xpath_one(page, '//img[@rel="lightbox"]').attrib['src']
-        infoblk = xpath_one(page, '//td/b[contains(text(), "CAUCUS/DELEGATION MEMBERSHIP")]')
+        infoblk = xpath_one(
+            page, '//td/b[contains(text(), "CAUCUS/DELEGATION MEMBERSHIP")]')
         infoblk = infoblk.getparent()
-        info = infoblk.text_content()
         cty = xpath_one(infoblk, "./b[contains(text(), 'ASSIGNMENTS')]")
         cty = cty.getnext()
 
-        partyblk = filter(lambda x: "District" in x,
-                          page.xpath('//p[@align="center"]//text()'))[0]
-
         party_flags = {
             "Democrat": "Democratic",
             "Republican": "Republican",
             "Independent": "Independent"
         }
-
-        if leg_info['name'].startswith("Vacant"):
-            return
-
-        party = 'other'
-        for p in party_flags:
-            if p in partyblk:
-                party = party_flags[p]
-
-        if party == 'other':
-            raise Exception
+        party_info = page.xpath(
+            '//div[@class="FullName"]/ancestor::td[1]/text()')
+        (party_info, ) = [x.strip() for x in party_info if x.strip()]
```

Comment:
The legislators scrape fails for me on this line as soon as it starts hitting house pages.

http://house.louisiana.gov/H_Reps/members.asp?ID=98

with this error:
ValueError: need more than 0 values to unpack